code/modules/paperwork/pen.dm

```diff
@@ -137,17 +137,24 @@
  */
 /obj/item/pen/edagger
 	origin_tech = "combat=3;syndicate=1"
-	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut") //these wont show up if the pen is off
 	var/on = 0
 	var/brightness_on = 2
 	light_color = LIGHT_COLOR_RED
 
+/obj/item/pen/edagger/attack(mob/living/M, mob/living/user, def_zone)
+	if(on == 1 && user.dir == M.dir && !M.weakened && user != M)
+		M.apply_damage(10, BRUTE, BODY_ZONE_CHEST)
+		M.Weaken(1)
+		M.visible_message("<span class='warning'>[user] stabs [M] in the back!", "<span class='userdanger'>[user] stabs you in the back! The energy blade makes you collapse in pain!")
+	. = ..()
```

Comment:
```suggestion
	return ..()
```
Standard is to return the parent call instead of set the return value to the parent call.